redcap_bridge/utils.py

```diff
@@ -46,4 +49,34 @@ def get_repo_state(path):
         commit_hash = ''
         clean = False
 
-    return commit_hash, clean
\ No newline at end of file
+    return commit_hash, clean
+
+
+def compressed_record(csv_file, compressed_file=None):
+    custom_csv = pd.read_csv(csv_file)
+    df = pd.DataFrame(custom_csv)
+    custom_df = df.filter(regex='.___.')
+    for column in custom_df:
+        if 1 in custom_df[column].values:
+            style_moda = re.search('___(.+?)', column).group(1)
+            custom_df[column] = style_moda
+        else:
+            del custom_df[column]
+
+    list_column = custom_df.columns.tolist()
+
+    for i, item in enumerate(list_column):
+        print(item)
+        name = re.search('(.+?)___', item).group(1)
+        if i != 0:
+            get_previous = list_column[i-1]
+            if name in item and name in get_previous:
+                custom_df[name] = custom_df[[name, item]].agg(','.join, axis=1)
+                del custom_df[item]
+            else:
+                custom_df[name] = custom_df[[item]].agg(','.join, axis=1)
```

Comment:
You are still running `.agg` on a single column of the dataframe. Are you sure this is contributing to the functionality?